client/app/(onboarding)/home/page.tsx

```diff
@@ -26,8 +15,65 @@ e.g. app\(onboarding)\home maps to route: http://localhost:3000/home
 */
 
 export default function Page() {
+  const fontSize = "text-2xl";
+  const headerStyle = "text-bc-bg-blue text-2xl";
+  const tableBorder = "border border-solid border-bc-bg-dark-grey";
+
+  const handleIdirLogin = () => {
+    signIn("keycloak", undefined, { kc_idp_hint: "idir" });
+  };
+
+  const handleBceidLogin = () => {
+    signIn("keycloak", undefined, { kc_idp_hint: "bceidbusiness" });
+  };
+
+  const handleBceidSignupClick = async () => {
+    // We don't have a build solution for the env variables in the frontend yet,
+    // so we need to get the env value from the backend
+    const env = await getEnvValue("NODE_ENV");
+    if (env === "production") {
+      window.open(
+        "https://www.bceid.ca/register/business/",
+        "_blank",
+        "noopener,noreferrer",
+      );
+    } else {
+      window.open(
+        `https://www.${env}.bceid.ca/register/business/`,
+        "_blank",
+        "noopener,noreferrer",
+      );
+    }
+  };
```

Comment:
There was a big discussion in the dev channel about this. Shon made a helpful action to get env vars to the front end since we don't have a good way to build public frontend env vars into our next app. It might require 3 separate docker images for each environment. This is the solution for now, I also added some protection to the action so we can only get whitelisted env vars.